src/ios/PurchasesPlugin.swift

```diff
@@ -12,11 +12,11 @@ import RevenueCat
 
 public class CDVPurchasesPlugin : CDVPlugin {
 
-    typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
+    public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
```

Comment:
It doesn't really matter because no one should be using this stuff directly, rather they should be using the Typescript methods. 
But our of curiosity, why does this need to be public now?